app/views/home/index.html.erb

```diff
@@ -1,48 +1,37 @@
-<div id="page-top">
-<!-- Navigation-->
+<div id="top">
 <nav class="navbar navbar-expand-lg navbar-light fixed-top shadow-sm" id="mainNav">
   <div class="container px-5">
-    <img src="/Ico6.png"
-    <%= image_tag("Yoyaku-02 1.png", :alt => "rss feed", :class => "ico-yoyaku") %>
-    <a class="navbar-brand fw-bold" href="#page-top">Yoyaku</a>
+    <a href="#top">
```

Comment:
I believe we don't need this new anchor, wrapping both title and image with the existing anchor should be enough